README.md

```diff
@@ -44,15 +44,22 @@ If you build this project from a git working directory, the main [CMakeLists.txt
 
 ## Drivers officially supported architectures
 
-Right now our drivers (kernel module, eBPF probe) officially support two main architectues: `ARM64` and `x86_64`.
+Right now our drivers officially support the following architectures:
 
 ### x86_64
```

Comment:
I think it could be the right moment to introduce a table for this :) 

|  arch | kmod  | eBPF   |
|---|---|---|
| x86_64| >= 2.6 | >= 4.14 |
|aarc64| >= 3.4 | >= 4.17 |
| s390x | >= 2.6 | :x:  |

Reply:
Moreover i think a little note about s390x eBPF could help too!